as/src/Makefile

```diff
@@ -95,7 +95,9 @@ INCLUDES += -I$(COMMON)/target/$(PLATFORM)/include
 INCLUDES += -I$(MOD_LUA)/target/$(PLATFORM)/include
 INCLUDES += -I$(JANSSON)/src
 INCLUDES += -I$(S2)
-INCLUDES += -I$(XDR_INCLUDES)
+ifneq ($(XDR_INCLUDES), )
```

Comment:
Something like `ifdef` would be a bit more direct:
<pre>
ifdef S2
  INCLUDES += -I$(S2)
endif
ifdef XDR_INCLUDES
  INCLUDES += -I$(XDR_INCLUDES)
endif
</pre>

Reply:
done